apps/shipments/geojson.py

```diff
@@ -116,28 +144,46 @@ def get_linestring_list(tracking_points):
     @staticmethod
     def get_linestring_feature(tracking_points):
         try:
+            LOG.debug(f'Device tracking get_linestring_list with tracking points {tracking_points}.')
+            log_metric('transmission.info', tags={'method': 'get_linestring_list'})
+
             linestring, linestring_timestamps = DeviceTrackingPoint.get_linestring_list(tracking_points)
             return Feature(geometry=linestring, properties={"linestringTimestamps": linestring_timestamps})
 
         except Exception:
+            LOG.error(f'Device tracking get_linestring_feature exception {Exception}.')
```

Comment:
This isn't going to print out anything but the Exception class. If you want to capture the details of the exception, you have to catch it like this:
`except Exception as ex:`
and then print out `ex`.